### ISSUE TITLE
Show contrast ratio with pure black of user-selected RGB color code

### ISSUE THREAD

**Issue by masakudamatsu** [masakudamatsu/luminance-picker#8]
Convert the user-selected RGB color code into its contrast ratio to pure black
- Relative luminance is not straightforward to understand
- What the user cares about is the contrast ratio to observe the WCAG 2.1 guideline. 
- The contrast ratio of two colors is easy to calculate if each color's contrast ratio to pure black is known.

Each of the RGB color code fields is identified with R, G, and B.
- Follows Sketch app's color picker window:
<img width="222" alt="Screenshot 2020-08-24 at 11 48 50" src="https://user-images.githubusercontent.com/36348559/90998780-dfa86e00-e5ff-11ea-95b1-c055cc3172ae.png">

The color contrast value is identified with "Contrast Ratio with Pure Black"

The conversion follows the formula in https://www.w3.org/TR/WCAG20-TECHS/G17.html

Use
```
<input type="text" inputmode="decimal"  pattern="[0-9]*"/>
```
for each of the R, G, B values

References
- https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/ (for why type="number" should be avoided)
- https://css-tricks.com/better-form-inputs-for-better-mobile-user-experiences/ (for the use of decimal inputmode)

**Comment by masakudamatsu** [comment]
We didn't deal with the cases where each value of the RGB code (normalized to be between 0 and 1) is less than 0.03928.
